src/core/sys/posix/signal.d

```diff
@@ -1427,6 +1437,18 @@ else version (Solaris)
         ___data __data;
     }
 
+    enum SI_NOINFO  = 32767;
+    enum SI_DTRACE  = 2050;
+    enum SI_RCTL    = 2049;
+    enum SI_USER    = 0;
+    enum SI_LWP     = -1;
+    enum SI_QUEUE   = -2;
+    enum SI_TIMER   = -3;
+    enum SI_ASYNCIO = -4;
+    enum SI_MESGQ   = -5;
+
+    enum SIGIO = SIGPOLL;
```

Comment:
I'd move this with the others, but don't think it would be harming anything at this point in time.